### ISSUE TITLE
Python module 'funcsigs' is needed, but not installed as part of a inferpy package

### ISSUE THREAD

**Issue by parsa-asgari** [PGM-Lab/InferPy#112]
long story short:
>$pip install inferpy<  does not install python's funcsigs library.

It is suggested to include it as part of the yml or whatever list pip is receiving to install inferpy.

**Comment by VertigoX64** [comment]
Here's the fix i've made in my fork:
[](https://github.com/PGM-Lab/InferPy/compare/master@%7B1day%7D...VertigoX64:master)

**Comment by rcabanasdepaz** [comment]
Yes, this is the way it is supposed to be fixed. If you prefer, you can do a pull request to our develop branch. Otherwise, I can fix it myself.

Thanks!!
